### PR TITLE
[IAC-1472] - Revamp  GhA report 

### PR DESCRIPTION
**Pull request by daianamezdrea**

### CODE REVIEW

File: github_action_report.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        var e = document.getElementById(id)
+        if (e.style.display == 'none')
+            e.style.display = 'table-row-group';
+        else
+            e.style.display = 'none';
+
+    }
+
+    //  Google Analytics
+    (function(i, s, o, g, r, a, m) {
```
Comment (carabasdaniel) [inline github_action_report.js]: Please be explicit with the function parameters here

File: github_actions_report.html.erb [hunk 2]
```diff
@@ -1,80 +1,103 @@
...
+  <script src='./web_libraries/sorttable.js'></script>
+  <link rel='stylesheet' href='./web_libraries/bootstrap.min.css'>
+  <script src='./web_libraries/DataTables/datatables.js'></script>
+  <link rel='stylesheet' href='./web_libraries/DataTables/datatables.css'>
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+  <script src="github_action_report.js"></script>   
+  <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+  <script type="text/javascript">
+  google.charts.load("current", {packages:["corechart"]});
```
Comment (carabasdaniel) [inline github_actions_report.html.erb]: Can this script be moved into the github_action_report.js file ?

File: github_actions_report.html.erb [hunk 3]
```diff
@@ -1,80 +1,103 @@
...
-
-ga('create', 'UA-157301458-1', 'auto');
-ga('send', 'pageview', { 'page': location.pathname + location.search + location.hash});
-ga('set', 'anonymizeIp', true);
-</script>
-<!-- End Google Analytics -->
 </head>
 <body>
 <h1 style="text-align:center;">Github Actions Report</h1>
+<center><div id="donutchart" style="width: 900px; height: 500px;"></div></center>
```
Comment (carabasdaniel) [inline github_actions_report.html.erb]: using a fixed dimension might cause a couple of issues, can you use relative dimensions here ?

File: github_actions_report.html.erb [hunk 4]
```diff
@@ -1,80 +1,103 @@
...
+<br>
+<input type="text" id="myInput" onkeyup="myFunction()" placeholder="Search for names.." title="Type in a name">
```
Comment (carabasdaniel) [inline github_actions_report.html.erb]: Please rename myFunction to a relevant name

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+    text-align: left;
+    padding: 8px;
+    position: relative;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+#myInput {
```
Comment (carabasdaniel) [inline styles.css]: please rename myInput to a relevant name

**Review by carabasdaniel — CHANGES_REQUESTED** [review]

File: github_action_report.js (hunk 1 above)
Comment (daianamezdrea) [reply github_action_report.js]: Hi Dani, according to : https://developers.google.com/analytics/devguides/collection/analyticsjs , this is how the function is written, do we want to change it in order to be more explicit?

**Pull request closed without merging**